letsencrypt/cli.py

```diff
@@ -636,8 +638,12 @@ def preprocess_args(self, args):
                 self.verb = token
                 return reordered
 
-        self.verb = "run"
-        return args + ["run"]
+        if "--standalone" in args and "--installer" not in args and "-i" not in args:
```

Comment:
So we make an exception for `--standalone`, are there going to be other exceptions in the future?

I just don't like the added complexity and prefer the approach described above. `run` is the default, full stop. If you run `letsencrypt --standalone`, we try to install if there is an installer available. If another webserver is running it won't work but as you pointed out to me in another PR, this is true regardless of whether `auth` or `run` is used. If, however, there is no installer available we error out quickly, explaining the problem and pointing them to the `auth` command. This behavior is fairly self documenting and I think will be more easily understood by our users than your approach.

We made the decision to make `run` the default and I think we should stick to that decision regardless of the flags given on the command line.